lib/alerts_viewer_web/live/bus_live.ex

```diff
@@ -150,7 +157,29 @@ defmodule AlertsViewerWeb.BusLive do
   def seconds_to_minutes(nil), do: nil
 
   def seconds_to_minutes(seconds) do
-    (seconds / 60) |> Float.round(1)
+    (seconds / 60) |> round
+  end
+
+  def display_minutes(assigns) do
+    list_of_minutes =
+      assigns.stats_by_route
+      |> RouteStats.vehicles_instantaneous_headway_secs(assigns.route)
```

Comment:
suggestion: I like the idea of making this component function, but with this function hardcoded here, doesn't that mean that we are getting the same results each time we call it? If I'm understanding right, maybe this function needs to be passed in so that e.g. `RouteStats.vehicles_schedule_adherence_secs ` and `RouteStats.vehicles_instantaneous_minus_scheduled_headway_secs` can be passed in for those columns?

Reply:
You're right, whoops